metabase/metabase
# Remove dead search bar

### Description

Ran across a dead search bar component that was disabled 8 years ago. 💀 

![Screen Shot 2023-03-22 at 12 58 14 PM](https://user-images.githubusercontent.com/30528226/227009272-dd4ddd35-561f-4e91-b7ff-95cf7339ab61.png)

Describe the overall approach and the problem being solved.

### How to verify

The only place this component is used is in the audit log where we preview non-SQL queries with the old query builder code.

1. Admin Settings > Audit > click on any ad hoc query.
2. [page crash]
3. exit admin settings
4. browser back button

We should replace all this mostly-unused query builder code at some point. see: https://github.com/metabase/metabase/issues/28639
